### PR TITLE
'xclip' as secondary clipboard utility

### PR DESCRIPTION
**Pull request by kxxvii**
Noticed that command-line copying didn't work when xsel was missing. So I added [xclip](https://github.com/astrand/xclip) as a secondary clipboard utility for free *nixes.

Tried it out on Debian GNU/Linux (Stretch), seems to work. If it doesn't for others I'd be glad to learn why that is and what could be done about it.

### CODE REVIEW

**Comment by jarun** [comment]
The patch looks fine. Can you please fix the lint issues so we can merge it?

File: ddgr [hunk 1]
```diff
@@ -1503,9 +1503,12 @@ class DdgCmd(object):
                     try:
                         # try copying the url to clipboard using native utilities
                         if sys.platform.startswith(('linux', 'freebsd', 'openbsd')):
-                            if shutil.which('xsel') is None:
+                            if shutil.which('xsel') != None:
```
Comment (jarun) [inline ddgr]: Should be `is not None`, lint complains.

Same for line 1508.

Comment (kxxvii) [reply ddgr]: Totally forgot about linting. Just pushed a commit.

**Comment by jarun** [comment]
Thank you! Would it be possible for you to port the changes from `googler` to use tmux and gnu screen clipboards?

The relevant code is here: https://github.com/jarun/googler/blob/master/googler#L2095

**Comment by kxxvii** [comment]
I _might_ be able to. I use tmux, but that doesn't really guarantee that I could implement its clipboard capabilities in Python, haha. Could take a stab at it and if I succeed you'll notice. ;)

Great idea though, I like it.

**Pull request merged**